openapi/openapi.go

```diff
@@ -74,17 +76,21 @@ func LoadFromYaml(openApiFilepath string) (*OpenAPI, error) {
 	return &api, err
 }
 
+// ResolveReferences traverses the entire openapi struct tree and loads te h
+// TODO: this doesn't have to be public
 func (o *OpenAPI) ResolveReferences(absoluteBasePath string) error {
 	o.SetBasePath(filepath.Dir(absoluteBasePath))
+	// TODO: what if JSON?
 	o, err := tree.Traverse(o, tree.ResolveRefs)
 	if err != nil {
 		return fmt.Errorf("tree traversal error: %w", err)
 	}
 	return nil
 }
 
-// Load loads the yaml content into the OpenAPI struct. Extraneous functions that aid in this process should live here.
-func (o *OpenAPI) Load(content []byte) error {
+// LoadFromBytes loads the content into an OpenAPI struct. Extraneous functions that aid in this process should live here.A
```

Comment:
`.A`?

Reply:
Vim mistake, I think. Will clean up in followup 👌